### PR TITLE
fix(orchestrator):improve workflow details card in workflow page

### PR DESCRIPTION
**Pull request by LiorSoffer**
resolve [FLPATH-1968](https://issues.redhat.com/browse/FLPATH-1968)
image

### CODE REVIEW

**Comment by batzionb** [comment]
@LiorSoffer 
Can you add a screenshot please?

File: workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx [hunk 1]
```diff
@@ -87,27 +87,27 @@ const WorkflowDefinitionDetailsCard = ({
...
+              label="description"
+              value={formattedWorkflowOverview?.description}
+            >
+              {loading ? (
+                <Skeleton variant="text" />
+              ) : (
+                formattedWorkflowOverview?.description
+              )}
+            </AboutField>
+          </Grid>
```
Comment (gciavarrini) [inline workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx:110]: are these changes needed?

Comment (batzionb) [reply workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx:110]: @gciavarrini 
WDYM? How would you make the change in the screen otherwize?

File: workspaces/orchestrator/.changeset/big-needles-grin.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@red-hat-developer-hub/backstage-plugin-orchestrator': patch
+---
+
+improve workflow details card in workflwo page
```
Comment (batzionb) [inline workspaces/orchestrator/.changeset/big-needles-grin.md:5]:
```suggestion
improve workflow details card in workflow page
```

File: workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx (hunk 1 above)
Comment (gciavarrini) [reply workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx:110]: @batzionb 
I'm ok with the changes, but it seems that those lines were just indented differently. That's why I'm asking if it was intended. 
Not a "blocker" comment :)

File: workspaces/orchestrator/.changeset/big-needles-grin.md (hunk 2 above)
Comment (gciavarrini) [reply workspaces/orchestrator/.changeset/big-needles-grin.md]: Same typo in the PR title :)

File: workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx [hunk 3]
```diff
@@ -87,27 +87,27 @@ const WorkflowDefinitionDetailsCard = ({
   return (
     <InfoCard title="Details" className={classes.details}>
       <Grid container spacing={3} alignContent="flex-start">
-        <Grid container item md={4} spacing={3} alignContent="flex-start">
+        <Grid container item md={12} spacing={3} alignContent="flex-start">
```
Comment (batzionb) [inline workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx:90]: There is no need anymore for two hierarchies of the grid, you can move the inner one

File: workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx (hunk 1 above)
Comment (batzionb) [reply workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx:110]: if you look carefully you see it's not just an indent change
The </Grid> close tag was moved down

Comment (gciavarrini) [reply workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowPage/WorkflowDetailsCard.tsx:110]: ohh! ok now i see it. Thanks
